tf-modules/distribution/main.tf

```diff
@@ -31,11 +33,31 @@ resource "aws_secretsmanager_secret_version" "thin_egress_urs_creds" {
   })
 }
 
+module "tea_map_cache" {
+  prefix = var.prefix
+  source = "../tea-map-cache"
+  lambda_processing_role_arn = var.lambda_processing_role_arn
+  tea_api_url = module.thin_egress_app.internal_api_endpoint
+  tags = var.tags
+  lambda_subnet_ids = var.subnet_ids
+  vpc_id = var.vpc_id
+}
```

Comment:
nit: align `=` signs for consistency

Reply:
Fair, good nit.   I updated a few others in the file.   I wish .tf files had a 'prettier' or other autoformat ruleset.  

a818be460